src/boot/axios.js

```diff
@@ -26,20 +26,34 @@ let zapUpdateExceptions = (payload, statusCode, message) => {}
 // You can set this to false to not log all the roundtrips
 const log = false
 
+// Get the search string from the URL
 let search = window.location.search
 
+// Remove the '?' if present
 if (search[0] === '?') {
   search = search.substring(1)
 }
-let query = querystring.parse(search)
-if (window.sessionStorage.getItem('session_uuid') == null) {
+
+// Parse the query string using URLSearchParams for browser environments
+let query = new URLSearchParams(search)
+
+// Extract stsApplicationId and trim it
+let stsApplicationId = query.get('stsApplicationId')?.trim()
+
+// Get the current session UUID from sessionStorage
+let currentSessionUuid = window.sessionStorage.getItem('session_uuid')
+
+// If stsApplicationId exists, update the session_uuid
+if (stsApplicationId) {
```

Comment:
This if will always be true in Studio, right? But if this is true, then the else if on the line 53 will never even get to be tested.

So if you are in Studio, when do you even create the UUID?

Reply:
fixing